_posts/2019-08-26-how-to-use-gruntwork-service-catalog.adoc

```diff
@@ -1,23 +1,15 @@
-[[how_to_use_gruntwork_service_catalog]]
+---
+categories: Docker
+image: /assets/img/guides/service-catalog/grunty-blocks.png
+excerpt: Learn about production-grade infrastructure, Terraform, Terragrunt, Packer, Docker, immutable infrastructure, versioning for infrastructure code, automated tests for infrastructure code, and more.
+permalink: /guides/:categories/:title
```

Comment:
Do we need to specify a permalink for every post? Seems error prone. Could we define this in the layout or Jekyll config instead?

Reply:
Yes we can define this in the config